dpypelines/pipeline/shared/utility.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+def enrich_online(message_creating_function):
+    def wrapper(*args):
+        msg: str = message_creating_function(*args)
+        
+        #Get the enviormentl variable
+        enrich_message = os.environ.get('ENRICH_OUTGOING_MESSAGES', None)
+        #If there is a value stored add (enrich) message otherwise return original message
+        if enrich_message is not None:
+            return msg + enrich_message
```

Comment:
can you add a line break between the original and added message please. It'll be a url so fresh line will probably be more readible.